cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -396,7 +397,14 @@ private Future<Void> reconcile(Reconciliation reconciliation, String host,
                                    CruiseControlApi apiClient, KafkaRebalance kafkaRebalance,
                                    KafkaRebalanceState currentState, KafkaRebalanceAnnotation rebalanceAnnotation) {
 
-        LOGGER.infoCr(reconciliation, "Rebalance action from state [{}]", currentState);
+
+        if (kafkaRebalance != null && kafkaRebalance.getStatus() != null
+                && Objects.equals(rebalanceStateConditionType(kafkaRebalance.getStatus()), "Ready")
+                && rawRebalanceAnnotation(kafkaRebalance) == null) {
+            LOGGER.infoCr(reconciliation, "Rebalancing is completed. You can use the  `refresh` annotation to ask for a new rebalance request", currentState);
```

Comment:
@scholzj To handle the case if Rebalancing is complete and the resource is still there, I have introduced this `if` block which prompts the user that `Rebalancing is complete` and they can request a new rebalance if they want